src/Symfony/Component/Cache/Adapter/PdoAdapter.php

```diff
@@ -0,0 +1,380 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Cache\Adapter;
+
+use Symfony\Component\Cache\Exception\InvalidArgumentException;
+
+class PdoAdapter extends AbstractAdapter
+{
+    /**
+     * {@inheritdoc}
+     */
+    protected $maxIdLength = 255;
```

Comment:
This anticipates  #19521 (esp. https://github.com/symfony/symfony/pull/19521/files#diff-a5185cd58702e8e073786794a423cb27R404)